Source/WebKit/GPUProcess/graphics/WebGPU/RemoteQueue.messages.in

```diff
@@ -32,10 +32,10 @@ messages -> RemoteQueue Stream {
     void Destruct()
     void Submit(Vector<WebKit::WebGPUIdentifier> commandBuffers)
     void OnSubmittedWorkDone() -> ()
-    void WriteBufferWithCopy(WebKit::WebGPUIdentifier identifier, WebCore::WebGPU::Size64 bufferOffset, Vector<uint8_t> data)
+    void WriteBufferWithCopy(WebKit::WebGPUIdentifier identifier, WebCore::WebGPU::Size64 bufferOffset, GPURemoteQueueOpaqueData data)
```

Comment:
would seem like this name should either include the term `Buffer` to match the `Texture` variant on line 38 below or the `Texture` variant could be removed and both messages use the same type